recipe/build.sh

```diff
@@ -96,35 +96,24 @@ for f in $PETSC_ARCH/lib/petsc/conf/petscvariables $PETSC_ARCH/lib/pkgconfig/PET
   fi
 done
 
-sedinplace() {
-  if [[ $(uname) == Darwin ]]; then
-    sed -i "" "$@"
-  else
-    sed -i"" "$@"
-  fi
-}
-
 # Remove abspath of ${BUILD_PREFIX}/bin/python
-sedinplace "s%${BUILD_PREFIX}/bin/python%python%g" $PETSC_ARCH/include/petscconf.h
-sedinplace "s%${BUILD_PREFIX}/bin/python%python%g" $PETSC_ARCH/lib/petsc/conf/petscvariables
-sedinplace "s%${BUILD_PREFIX}/bin/python%/usr/bin/env python%g" $PETSC_ARCH/lib/petsc/conf/reconfigure-arch-conda-c-opt.py
+sed -i.bak "s%${BUILD_PREFIX}/bin/python%python%g" $PETSC_ARCH/include/petscconf.h
+sed -i.bak "s%${BUILD_PREFIX}/bin/python%python%g" $PETSC_ARCH/lib/petsc/conf/petscvariables
+sed -i.bak "s%${BUILD_PREFIX}/bin/python%/usr/bin/env python%g" $PETSC_ARCH/lib/petsc/conf/reconfigure-arch-conda-c-opt.py
 
 # Replace abspath of ${PETSC_DIR} and ${BUILD_PREFIX} with ${PREFIX}
 for path in $PETSC_DIR $BUILD_PREFIX; do
     for f in $(grep -l "${path}" $PETSC_ARCH/include/petsc*.h); do
         echo "Fixing ${path} in $f"
-        sedinplace s%$path%\${PREFIX}%g $f
+        sed -i.bak s%$path%\${PREFIX}%g $f
     done
 done
 
 make
 
 
 if [[ "$CONDA_BUILD_CROSS_COMPILATION" != "1" ]]; then
-  # FIXME: Workaround mpiexec setting O_NONBLOCK in std{in|out|err}
-  # See https://github.com/conda-forge/conda-smithy/pull/337
-  # See https://github.com/pmodels/mpich/pull/2755
-  make check MPIEXEC="${RECIPE_DIR}/mpiexec.sh"
+  make check
```

Comment:
@mirk Perhaps we should try MPICH again for the infamous issue that forced us to use this mpiexec hack.

Reply:
The fix seems to have been merged to mpich, is mpiexec.sh still necessary?